download/community.md

```diff
@@ -24,6 +24,4 @@ Flyway is available in many different forms. Choose from the options below for s
     {% include download.html name="Gradle" icon="cogs" url="/documentation/usage/gradle/#installation" %}
 </div>
 
-Looking for Flyway Teams, or interested in what powerful features are available to enhance your workflows? See a comparison of the versions, begin a free trial, or purchase a license in [download / pricing](/download).
-
-{% include trialpopup.html %}
```

Comment:
What was the rationale for removal? What was the prior behaviour for this?

Reply:
The trial popup include adds a form that is spawned by the button on the download page. So it should be on the community download page (it was there due to a copy/paste error)